docs/components_page/components/navbar/__init__.py

```diff
@@ -0,0 +1,95 @@
+from pathlib import Path
+
+import dash_core_components as dcc
+import dash_html_components as html
+
+from ...api_doc import ApiDoc
+from ...helpers import (
+    ExampleContainer,
+    HighlightedSource,
+    load_source_with_environment,
+)
+from ...metadata import get_component_metadata
+from .navbar import navbar
+
+GITHUB_EXAMPLES = "https://github.com/ASIDataScience/dash-bootstrap-components/blob/master/examples/advanced-component-usage/CustomNavbar.py"  # noqa
```

Comment:
I changed this to link directly to the example. As it stands, this link will break because the file isn't there yet. Once this is merged into master it should all be fine 🤞